code/datums/components/pellet_cloud.dm

```diff
@@ -124,8 +124,11 @@
 		shell.loaded_projectile.wound_bonus = original_wb
 		shell.loaded_projectile.bare_wound_bonus = original_bwb
 		pellets += shell.loaded_projectile
-		if(!shell.throw_proj(target, targloc, shooter, params, spread))
+		var/turf/curloc = get_turf(user)
```

Comment:
`current_loc`--stop abbreviating names, I already don't like "targloc"

Reply:
Done , renamed the vars.